truenas_install/__main__.py

```diff
@@ -184,9 +184,13 @@ def enable_user_services(root, old_root):
                     os.makedirs(f"{root}/boot/efi", exist_ok=True)
                     for disk in disks:
                         run_command(["chroot", root, "grub-install", "--target=i386-pc", f"/dev/{disk}"])
-
-                        run_command(["chroot", root, "mkdosfs", "-F", "32", "-s", "1", "-n", "EFI", f"/dev/{disk}2"])
-                        run_command(["chroot", root, "mount", "-t", "vfat", f"/dev/{disk}2", "/boot/efi"])
+                        # Check if nvme disk and instead use nvme naming convention.
+                        if ("nvme" in disk):
```

Comment:
`(` and `)` are redundant here

Reply:
`                        # Check if nvme disk and instead use nvme naming convention.
                        if "nvme" in disk: partition = "{disk}p2"
                        else: partition = "{disk}2"
                        run_command(["chroot", root, "mkdosfs", "-F", "32", "-s", "1", "-n", "EFI", f"/dev/{partition}"])
                        run_command(["chroot", root, "mount", "-t", "vfat", f"/dev/{partition}", "/boot/efi"])`

Is this better?

Reply:
Yes!